### ISSUE TITLE
Device not certified even with new update

### ISSUE THREAD

**Issue by Sams0ngenjoyer** [chiteroman/PlayIntegrityFix#398]
I am usign one ui 6 android 14 rom and today when i tried to buy something with google pay it showed up this device cant make tap payments i looked up if my device was certified and it was! I saw that there was an update the 17.0 and still nothin can yall tell me what to do becouse i rea
![Screenshot_20240815_184851_Google Play Store](https://github.com/user-attachments/assets/1e1f1a8c-e33d-477d-8cee-e54caa5d437d)
![Screenshot_20240815_184843](https://github.com/user-attachments/assets/45896df3-5ce0-490d-8dcc-1b0bdbf18eaf)
lly need to have google pay working

**Comment by punkesta** [comment]
I have the same issue

**Comment by Sams0ngenjoyer** [comment]
Dang bro they need to fix this

**Comment by Sams0ngenjoyer** [comment]
Nothing works :((

**Comment by antonywu** [comment]
17.0 works just fine on my Pixel 7. Maybe the issue is that it is getting harder and harder to get custom roms to work with PI. Graphene OS already said they will move forward with a lawsuit against Google if Google doesn't offer a way to certify custom roms

**Comment by myroad94** [comment]
In Pixel 5, it used to work until yesterday or so. I updated to v17, clear data of play store, google service framework, rebooted. The play integrity test works for a few minutes. Then somehow it is not working anymore. I guess Google Play updated their way of detecting root?

**Comment by Zerogoki00** [comment]
Same problem on my Pixel 7 Pro with stock firmware

**Comment by NekoMoYi** [comment]
Same problem on my Pixel 6

**Comment by NOHEATED** [comment]
Same problem with Xiaomi Mi 8 with crDroid. ROM is signed and the kernel isn`t banned. Only basic one passes and custom fingerprint isn`t helping

**Comment by myroad94** [comment]
> In Pixel 5, it used to work until yesterday or so. I updated to v17, clear data of play store, google service framework, rebooted. The play integrity test works for a few minutes. Then somehow it is not working anymore. I guess Google Play updated their way of detecting root?

Yes, the new update 17.1 now fixes the issue... for now! I can see it changes the fingerprint :)

**Comment by myroad94** [comment]
@Sams0ngenjoyer please:
* only install magisk + this playintegrityfix (no zygisk assistant or shamiko or any other tools to hide root)
* enable zygisk in magisk, enforce denylist, and then add every google* application in denylist, including in system app, in particular, google services*, google play, google wallet
* stop force application and delete data of google play store, google wallet, google play services, google services framework (system app)
* reboot
* test play integrity fix with some app in google play

**Comment by Sams0ngenjoyer** [comment]
It works! I can add cards but.... the device shows as not certified?????🫠🫠

**Comment by Sams0ngenjoyer** [comment]
How the f does it work while it shows not certified🤔

**Comment by nvn152** [comment]
> How the f does it work while it shows not certified🤔

Wipe data of playstore and reboot. It worked for me.

**Comment by JustGuardian** [comment]
I get the same behaviour, is it possible that flasing a custom kernel is the reason of this?
